### PR TITLE
Button support in inline toolbar

### PR DESCRIPTION
**Pull request by wojtekw92**
## What is the purpose of this pull request?

new feature

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](http://docs.ckeditor.com/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](http://docs.ckeditor.com/#!/guide/dev_tests) and
[how to create tests](http://docs.ckeditor.com/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## What changes did you make?

*Give an overview…*

### CODE REVIEW

**Comment by wojtekw92** [comment]
@f1ames as mentioned earlier `CKEDITOR.inlineToolbar` is subject of another task from user story. This PR is only about positioning and and `inlineToolbarView`.

File: plugins/inlinetoolbar/plugin.js [hunk 1]
```diff
@@ -110,6 +122,61 @@
...
+			};
+
+			/**
+			 * Adds an item from the specified definition to the inline toolbar.
+			 *
+			 * @method
+			 * @param {String} name The menu item name.
+			 * @param {Object} definition The menu item definition.
+			 */
+			CKEDITOR.ui.inlineToolbarView.prototype.addMenuItem = function( name, definition ) {
```
Comment (mlewand) [inline plugins/inlinetoolbar/plugin.js]: I have asked this to be fixed in https://github.com/ckeditor/ckeditor-dev/pull/992#discussion_r142139001

File: plugins/inlinetoolbar/plugin.js [hunk 2]
```diff
@@ -24,7 +24,12 @@
 		} );
 		CKEDITOR.ui.balloonPanel.call( this, editor, definition );
 		/**
-		 * The editor for this balloon panel.
+		 * Registred menu items in inlineToolbarView
+		 * @private
+		 */
+		this.menuItems = [];
```
Comment (mlewand) [inline plugins/inlinetoolbar/plugin.js]: Again I have asked to fix this in https://github.com/ckeditor/ckeditor-dev/pull/992#discussion_r142140769

File: plugins/inlinetoolbar/plugin.js [hunk 3]
```diff
@@ -25,6 +25,7 @@
 		} );
 		CKEDITOR.ui.balloonPanel.call( this, editor, defParams );
 		this.listeners = [];
+		this.menuItems = [];
```
[the remark this answers is not in the document]
Comment (wojtekw92) [reply plugins/inlinetoolbar/plugin.js]: this property is moved in 2996(context task)

**Comment by mlewand** [comment]
There's a typo in the last commit message "Insted" it should fixed.

**Comment by mlewand** [comment]
This pull request was automatically closed as I made a cleanup with the branches (see https://github.com/ckeditor/ckeditor-dev/pull/957#issuecomment-336098618). Thus @wojtekw92 needs to make a follow up PR based on the correct feature branch.

**Pull request closed without merging**